lib/rules/no-jsx-attributes.js

```diff
@@ -0,0 +1,85 @@
+import Rule from './_base.js';
+
+let outro = `In HTML, all attributes are valid, but 'className' doesn't do anything.`;
+
+function message(name) {
+  switch (name) {
+    case 'className': {
+      return `Attribute, ${name}, does not assign the 'class' attribute as it would in JSX.  To assign the 'class' attribute, set the 'class' attribute, instead of 'className'. ${outro}`;
+    }
+
+    default: {
+      return `Incorrect html attribute name detected - "${name}", is probably unintended. Attributes in HTML are kebeb-case.`;
+    }
+  }
+}
+
+const fixMap = {
+  acceptCharset: null, // it is property
+  mediaGroup: null, // it is property
+  srcSet: null, // it is property
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset
```suggestion
  srcSet: 'srcset',
```

Reply:
updated